elm/src/Test/Runner/Node.elm

```diff
@@ -309,11 +309,52 @@ init { processes, globs, paths, fuzzRuns, initialSeed, report, runners } _ =
     ( model, Cmd.none )
 
 
+{-| The implementation of this function will be replaced in the generated JS
+with a version that returns `Just value` if `value` is a `Test`.
+
+If you rename this function you also need to update the regex that looks for it.
+
+-}
+check : a -> Maybe Test
+check value =
+    Nothing
```

Comment:
```suggestion
check =
    \value -> check value
```

I think a crash is better than silently failing if somehow this function is not replaced in the js.

Reply:
Wouldn’t `Debug.todo "Informative message"` be better?

Reply:
I forget that we are allowed to use `Debug.todo`. Yes it would!

Reply:
This makes the `.replace()` a bit more difficult but I’ll see what I can do.

Reply:
Now using Debug.todo